bundles/org.openhab.binding.surepetcare/README.md

```diff
@@ -158,108 +176,184 @@ Bridge surepetcare:bridge:bridge1 "Demo API Bridge" @ "SurePetcare" [ username="
+Switch                      SurePet_FlapLowBattery_1           "Flap Low Battery"                      <LowBattery>          (gSurePetcareFlap)       ["Energy", "LowBattery"]     {channel="surepetcare:flapDevice:api:CHANGE_ME:lowBattery"}
+Number:Dimensionless        SurePet_FlapBatteryLevel_1         "Flap Battery Level"                    <Battery>             (gSurePetcareFlap)       ["Measurement", "Energy"]    {channel="surepetcare:flapDevice:api:CHANGE_ME:batteryLevel", stateDescription="%.0f %%", unit="%"}
+Number:ElectricPotential    SurePet_FlapBatteryVoltage_1       "Flap Battery Voltage"                  <energy>              (gSurePetcareFlap)       ["Point"]                    {channel="surepetcare:flapDevice:api:CHANGE_ME:batteryVoltage", stateDescription="%.1f V", unit="V"}
+Switch                      SurePet_FlapOnline_1               "Flap Online"                                                 (gSurePetcareFlap)       ["Point"]                    {channel="surepetcare:flapDevice:api:CHANGE_ME:online"}
+Number:Power                SurePet_FlapDeviceRSSI_1           "Flap Device RSSI"                      <qualityofservice>    (gSurePetcareFlap)       ["Point"]                    {channel="surepetcare:flapDevice:api:CHANGE_ME:deviceRSSI", stateDescription="%.2f dBm", unit="dBm"}
+Number:Power                SurePet_FlapHubRSSI_1              "Flap Hub RSSI"                         <qualityofservice>    (gSurePetcareFlap)       ["Point"]                    {channel="surepetcare:flapDevice:api:CHANGE_ME:hubRSSI", stateDescription="%.2f dBm", unit="dBm"}
+
+// Equipment representing thing:
+// surepetcare:feederDevice:api:CHANGE_ME
+// (SurePetcare Feeder 1)
+Group                       gSurePetcareDevice1                  "SurePetcare Feeder 1"                                                        (gSurePetcareDevices)    ["Equipment"]
+Number                      SurePet_FeederId_1                   "Feeder ID"                                                                   (gSurePetcareDevice1)    ["Point"]                    {channel="surepetcare:feederDevice:api:CHANGE_ME:id"}
+String                      SurePet_FeederName_1                 "Feeder Name"                                                                 (gSurePetcareDevice1)    ["Point"]                    {channel="surepetcare:feederDevice:api:CHANGE_ME:name"}
+String                      SurePet_FeederProduct_1              "Feeder Product"                                                              (gSurePetcareDevice1)    ["Point"]                    {channel="surepetcare:feederDevice:api:CHANGE_ME:product"}
+Switch                      SurePet_FeederLowBattery_1           "Feeder Low Battery"                             <LowBattery>          (gSurePetcareDevice1)    ["Energy", "LowBattery"]     {channel="surepetcare:feederDevice:api:CHANGE_ME:lowBattery"}
```

Comment:
Perhaps it could be better aligned? See also below.

Reply:
Got it all aligned now.